lib/DualInlinePackage.tsx

```diff
@@ -96,7 +96,7 @@ export const Dip = ({
       })}
       <ChipBody
         width={bodyWidth}
-        length={(10 / 4) * numPinsOnEachSide}
+        length={numPinsOnEachSide * pitch + 0.5}
```

Comment:
on a plane but are you sure it’s not this? Would make more sense than adding 0.5mm

```suggestion
        length={(numPinsOnEachSide + 0.5) * pitch}
```

Reply:
<img width="1337" alt="Screenshot 2024-10-11 at 10 24 13 PM" src="https://github.com/user-attachments/assets/02e4e590-f5e4-43ae-8d4e-552f2b0a13dd">
According to this ref

Reply:
<img width="1008" alt="Screenshot 2024-10-11 at 10 28 42 PM" src="https://github.com/user-attachments/assets/877fbd06-9c48-49d4-8c64-ff002f42b997">
@seveibar 
oops! I think we have many types

Reply:
Makes sense